File: awesomestuff/src/main/java/com/awesome/dialog/AwesomeProgressDialog.java
package com.awesome.dialog;

import android.content.Context;

import com.awesome.R;

/**
 * Created by yarolegovich on 16.04.2016.
 */
public class AwesomeProgressDialog extends AwesomeBase<AwesomeProgressDialog> {

    public AwesomeProgressDialog(Context context) {
        super(context);
    }

    public AwesomeProgressDialog(Context context, int theme) {
        super(context, theme);
    }

    {
        setCancelable(false);
    }

    @Override
    protected int getLayout() {
        return R.layout.dialog_progress;
    }
}

File: fusion/src/main/java/com/awesome/dialog/AwesomeProgressDialog.kt
package com.awesome.dialog

import android.content.Context

import com.awesome.R

/**
 * Created by Sandip Savaliya on 16.04.2016.
 */
class AwesomeProgressDialog : AwesomeBase<AwesomeProgressDialog> {

    override val layout: Int
        get() = R.layout.dialog_progress

    constructor(context: Context) : super(context)

    constructor(context: Context, theme: Int) : super(context, theme)

    init {
        setCancelable(false)
    }
}
